src/wasm/wasm-ir-builder.cpp

```diff
@@ -483,6 +498,14 @@ Result<> IRBuilder::visitLoopStart(Loop* loop) {
   return Ok{};
 }
 
+Result<> IRBuilder::visitTryStart(Try* tryy, Name label) {
+  // The delegate label will be regenerated if we need it. See `visitDelegate`
+  // for details.
+  tryy->name = Name();
+  pushScope(ScopeCtx::makeTry(tryy, label));
```

Comment:
Yes; the `label` is the name used to branch to the `try` with normal branches like `br` and `br_if`. It will end up being the name of a block we wrap around the `try`. `tryy->name` will ultimately end up being another name used to delegate to this `try`. Unfortunately we can't reuse `label` for that because the wrapper block and the `try` must have different names to validate.